server/app/services/applicant/AnswerData.java

```diff
@@ -59,6 +59,12 @@ public static Builder builder() {
    */
   public abstract boolean isEligible();
 
+  /**
+   * True if the program's eligibility is set to gating. If false, tags that note that the applicant
```

Comment:
**suggestion:** If "true", right?

Reply:
No, if eligibilityIsGating == true, eligibility gates submission and all tags should be shown. If eligibilityIsGating == false, eligibility does NOT gate submission and 'not eligibile' tags should be hidden.

Reply:
Lol sorry - Friday brain. Yes!